addons/trenches/XEH_preInit.sqf

```diff
@@ -6,4 +6,23 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+[QGVAR(allowDigging), "CHECKBOX", [localize LSTRING(settingAllowDigging_displayName), localize LSTRING(settingAllowDigging_tooltip)], localize LSTRING(settingCategory), true] call CBA_Settings_fnc_init;
```

Comment:
Put into separate `initSettings.sqf` file and `#include "initSettings.sqf"` here. Also multiline them for readability.

Reply:
Is there a specific reason to create a new file for it? Since this file would be quite empty then

Reply:
Because every ace component does it like that.